modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/test/I18nServletTest.java

```diff
@@ -334,6 +336,45 @@ public void testSendRedirectWithoutContext() throws Exception {
 			mockHttpServletResponse.getHeader("Location"));
 	}
 
+	@Test
+	public void testServiceIfNotGroupFoundResponseForwardedToLocalized404Page()
```

Comment:
What about changing the test name to some like: `testResponseIsForwardedToLocalized404PageIfGroupNotFound`